node/config/def.go

```diff
@@ -208,6 +218,11 @@ func DefaultStorageMiner() *StorageMiner {
 			ExpectedSealDuration: Duration(time.Hour * 24),
 		},
 
+		PublishMsg: PublishMsgConfig{
+			PublishPeriod:  Duration(time.Hour),
+			MaxDealsPerMsg: 8,
+		},
```

Comment:
These are placeholder defaults for `PublishPeriod` and `MaxDealsPerMsg`, open to suggestions for these values

Reply:
What is the MaxDealsPerMsg supported by the network ?

Reply:
i have seen up to 30 on the chain already -but this should be proper documented with the max values